algoliasearch-apache/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <artifactId>algoliasearch</artifactId>
+        <groupId>com.algolia</groupId>
+        <version>3.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>algoliasearch-apache</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <async-http-client.version>4.1.4</async-http-client.version>
+        <jsr305.version>3.0.2</jsr305.version>
+        <algoliasearch-core-version>3.0.0-SNAPSHOT</algoliasearch-core-version>
+    </properties>
+
+    <dependencies>
+        <!--Apache http client-->
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpasyncclient</artifactId>
+            <version>${async-http-client.version}</version>
+        </dependency>
+
+        <!--Algolia dependencies-->
+        <dependency>
+            <groupId>com.algolia</groupId>
+            <artifactId>algoliasearch-core</artifactId>
+            <version>${algoliasearch-core-version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.algolia</groupId>
+            <artifactId>algoliasearch-core</artifactId>
+            <version>${algoliasearch-core-version}</version>
+            <type>test-jar</type>
```

Comment:
add classifier too:
```xml
<classifier>tests</classifier>
```

Reply:
Just curious, why do we need the `classifier` tag? 

When we run `mvn deploy` we already have a `.jar` suffixed with `tests` : `algoliasearch-apache-3.0.0-beta-1-tests.jar` Am I missing something?

Reply:
Fixed!